cfgov/jinja2/v1/_includes/molecules/call-to-action.html

```diff
@@ -21,9 +21,8 @@
    settings.button.text:    The text to display on button.
 
    ========================================================================== #}
-
-<div class="m-call-to-action">
-    {% if value %}
+{% if value %}
```

Comment:
lets go with this approach stated here: https://github.com/cfpb/cfgov-refresh/pull/1192